app/forms/claims/claim/provider_form.rb

```diff
@@ -21,6 +21,22 @@ def persist
     end
   end
 
+  def edit_back_link
```

Comment:
Should this just be "back_path"?

```suggestion
  def back_path
```